src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/Address.sql

```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE [dbo].[Address]

```

Comment:
Can you remove the square brackets from `[dbo]`?

(`[Address]` should have them.)

Reply:
👍